Python_Backend/config.py

```diff
@@ -28,17 +28,32 @@
 use_i2c = not vars(args)["no_i2c"]
 
 # Determine if Real Arduinos or the fake interface should be used.
+
 if use_i2c:
     arduinos = arduino_interface.arduino_interface
 else:
     arduinos = arduino_interface.fake_arduino_interface
 
+arduinos = arduino_interface.fake_arduino_interface
+
+logger = logging.get_logger("Database connection")
+try:
+    dbclient = pymongo.MongoClient('mongodb+srv://watlock:general@cluster0.7s0cr.mongodb.net/myFirstDatabase?retryWrites=true&w=majority')
```

Comment:
this prob shdnt be out like this, put in env?

Reply:
oops forgot to change that, ty